rfc6962-bis.xml

```diff
@@ -481,26 +474,119 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
+        </t>
+        <t>
+          <spanx style="verb">item_extensions</spanx> is a vector of 0 or more item extensions. This vector MUST NOT include more than one extension with the same <spanx style="verb">item_extension_type</spanx>. The extensions in the vector MUST be ordered by the value of the <spanx style="verb">item_extension_type</spanx> field, smallest value first.
+        </t>
+      </section>
+      <section title="Merkle Tree Leaves" anchor="tree_leaves">
+        <figure>
+          <preamble>
+            The leaves of a log's Merkle Tree correspond to the log's entries (see <xref target="log_entries"/>). Each leaf is the <xref target="mht">leaf hash</xref> of a <spanx style="verb">TransItem</spanx> structure of type <spanx style="verb">x509_entry</spanx> or <spanx style="verb">precert_entry</spanx>, which in this version (v2) encapsulates a <spanx style="verb">TimestampedCertificateEntryDataV2</spanx> structure. Note that leaf hashes are calculated as HASH(0x00 || TransItem), where the hashing algorithm is specified in the log's metadata.
+          </preamble>
+          <artwork>
     opaque TBSCertificate&lt;1..2^24-1&gt;;
 
     struct {
+        uint64 timestamp;
         opaque issuer_key_hash[HASH_SIZE];
         TBSCertificate tbs_certificate;
-    } CertInfo;
-
+        SctExtension sct_extensions&lt;0..2^16-1&gt;;
+    } TimestampedCertificateEntryDataV2;</artwork>
+        </figure>
+        <t>
+          <spanx style="verb">timestamp</spanx> is the <xref target="RFC5905">NTP Time</xref> at which the certificate or precertificate was accepted by the log, measured in milliseconds since the epoch (January 1, 1970, 00:00), ignoring leap seconds.
+        </t>
+        <t>
+          <spanx style="verb">issuer_key_hash</spanx> is the HASH of the public key of the CA that issued the certificate or precertificate, calculated over the DER encoding of the key represented as SubjectPublicKeyInfo. This is needed to bind the CA to the certificate or precertificate, making it impossible for the corresponding SCT to be valid for any other certificate or precertificate whose TBSCertificate matches <spanx style="verb">tbs_certificate</spanx>.
+        </t>
+        <t>
+          <spanx style="verb">tbs_certificate</spanx> is the DER encoded TBSCertificate from either the <spanx style="verb">leaf_certificate</spanx> (in the case of an <spanx style="verb">X509ChainEntry</spanx>) or the <spanx style="verb">pre_certificate</spanx> (in the case of a <spanx style="verb">PrecertChainEntryV2</spanx>). (Note that a precertificate's TBSCertificate can be reconstructed from the issued certificate's TBSCertificate by redacting the domain name labels indicated by the redacted labels extension, and deleting the SCT list extension and redacted labels extension).
```

Comment:
We already have...
  TBSCertificate <xref target="RFC5280"/>
...earlier in the document.  Is that not sufficient?